src/device.js

```diff
@@ -160,6 +172,12 @@ if (exports.isMobile) {
     // Android then it's assumed to be an iOS device
     exports.isIOS = true;
   }
+
+  if (exports.isTablet) {
```

Comment:
you can put this outside isAndroid check.

isMiniTablet should work irrespective of the platform.